### PR TITLE
completed task 7 - POST /api/articles/:article_id/comments

### PR DESCRIPTION
**Pull request by Koalaphant**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -199,3 +200,81 @@ describe("GET /api/articles/:article_id/comments", () => {
...
+    };
+
+    return request(app)
+      .post("/api/articles/1/comments")
+      .send(newComment)
+      .expect(404)
+      .then((response) => {
+        expect(response.body.msg).toBe("username does not exist");
+      });
+  });
```
Comment (ghost) [inline __tests__/app.test.js:279]: Nicely handled. Up to you if you want to consider edge cases of more keys that aren't required.

**Review by ghost — COMMENTED** [review]
I'm not sure how your 404 for a valid article but not existing is working. I can't see you using a util function to check the database or a query. You shouldn't be relying on psql error codes for that as it totally fine for the article_id to not be in the comments table if an article is new a first comment wouldn't be in that table.
